src/Lorentz/Contracts/TreasuryDAO/Types.hs

```diff
@@ -27,7 +27,7 @@ data TreasuryDaoContractExtra = TreasuryDaoContractExtra
   , ceSlashDivisionValue :: Natural -- d
   , ceMinXtzAmount :: Mutez -- y
   , ceMaxXtzAmount :: Mutez -- z
-  , ceMaxProposalSize :: Natural --z
+  , ceMaxProposalSize :: Natural -- s_max
```

Comment:
good catch, it seems the same mistake was made in the comment inside the `Default` instance, would you mind fixing that as well?